server/src/test/java/org/elasticsearch/search/fetch/subphase/FieldFetcherTests.java

```diff
@@ -439,7 +439,25 @@ private static QueryShardContext createQueryShardContext(MapperService mapperSer
             .put(IndexMetadata.SETTING_INDEX_UUID, "uuid").build();
         IndexMetadata indexMetadata = new IndexMetadata.Builder("index").settings(settings).build();
         IndexSettings indexSettings = new IndexSettings(indexMetadata, settings);
-        return new QueryShardContext(0, indexSettings, null, null, null, mapperService, null, null, null, null, null, null, null, null,
-            null, null, null, emptyMap());
+        return new QueryShardContext(
```

Comment:
I believe I just replaced `mapperService` with `mapperService.snapshot()` but then I needed to fix the line length.